### DISCUSSION TITLE
[bug]: Double-barrelled Cardholder Name Invalidation in Core Methods

### DISCUSSION THREAD

**Discussion by ihojmanb** [mercadopago/sdk-js#116] [Q&A] [answered]
I recently had an issue with a customer payment. The payment failed when trying to create  the card token with `mercadopago.fields.createCardToken`:
```
{
code: 316,
message: "invalid parameter cardholderName"
}
```
My customer last name includes a hyphen (something common in Chile, as well as in other latin-american countries), a symbol considered as invalid by the mercadopago cardholder name validation. I will process this by myself in my front end, but I would consider this as a mercadopago bug, given the context of our latin american customers.

**Comment by icaldana** [comment, marked as answer]
Hello @ihojmanb!

We apologize for the inconvenience it has caused to your workflow. I've tested and validated this is happening indeed. 
Thank you so much for reporting it, we added an item in our roadmap to address this bug properly.

Thank you!

**Reply by icaldana** [reply]
@ihojmanb the fix was deployed today! You can already use hyphen normally. 😄
